ReactNative/js/components/ListItem.jsx

```diff
@@ -42,6 +39,7 @@ export default function ListItem({ url, title, onPress, label }) {
 
   const name = parse(url).domain;
 
+  /* eslint-disable prettier/prettier */
```

Comment:
This will only disable eslint, not prettier formatting right? So if you run prettier it will still reformat this part (unless prettier also understands eslint directives).

Reply:
IDK, this suppresses the VS Code errors. We don't enforce automatic prettier.